### ISSUE TITLE
Single object shorthand methods not routing correctly

### ISSUE THREAD

**Issue by mpint** [wheresrhys/fetch-mock#477]
Hello, I'm using `fetch-mock: ^8.0.0-alpha.11` and seem to have found a bug where single object shorthand methods aren't routing correctly.

Here, the GET route seems to erroneously handle PUT requests on the route.
```
fetchMock.config.overwriteRoutes = false
fetchMock
  .get({
    matcher: `express:${endpoint}`,
    response: responses.getResponse,
  })
  .put({
    matcher: `express:${endpoint}`,
    response: responses.putResponse,
  })
```

However, this works as expected.

```
fetchMock.config.overwriteRoutes = false
fetchMock
  .get(`express:${endpoint}`, responses.getResponse)
  .put(`express:${endpoint}`, responses.putResponse)
```

**Comment by wheresrhys** [comment]
Thanks for the bug report. I'll investigate when I get time

**Comment by wheresrhys** [comment]
Fixed in 8.0.3

Thanks again for the bugfix
